docs/source/examples/managed-jobs.rst

```diff
@@ -625,6 +625,28 @@ you can still tear it down manually with
 .. note::
   Tearing down the jobs controller loses all logs and status information for the finished managed jobs. It is only allowed when there are no in-progress managed jobs to ensure no resource leakage.
 
+To adjust the size of the jobs controller instance, see :ref:`jobs-controller-custom-resources`.
+
+
+Setup and best practices
+------------------------
+
+.. _static-creds:
+
+Using static credentials
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+Since the :ref:`jobs controller <jobs-controller>` is a long-lived instance that will manage other cloud instances, it's best to **use static credentials that do not expire**. If a credential expires, it could leave the controller with no way to clean up a job, leading to expensive cloud instance leaks.
+
+To use static credentials for the jobs controller, just make sure the static credentials are in use by SkyPilot. They will be automatically uploaded to the jobs controller. **If you're already using local credentials that don't expire, no action is needed!**
+
+To set up static credentials:
+
+- **AWS**: :ref:`Create a dedicated SkyPilot IAM user <dedicated-aws-user>`.
+- **GCP**: :ref:`Create a GCP service account <gcp-service-account>`.
+- **Other clouds**: Make sure you are using credentials that do not expire.
```

Comment:
We should have kubernetes' guide linked here as well?

Reply:
We don't have a good guide for Kubernetes. I think there's no consistent way to do this since authentication may vary for different deployments. The existing "service account" guide is not relevant - service accounts only apply intra-cluster.

Reply:
> I think there's no consistent way to do this

This is wrong. We can create static credentials for k8s service accounts. However, we don't have good docs on how to do this except for the main setup docs, which include things like creating namespaces (not relevant). I need to follow up on this.